06-llm-pr-review/backend/.env.example

```
@@ -0,0 +1,3 @@
+APP_NAME=FastAPI Template
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)